tox.ini

```diff
@@ -73,19 +73,17 @@ deps =
 
     # The oldestdeps factor is intended to be used to install the oldest versions of all
     # dependencies that have a minimum version.
-    oldestdeps: pyerfa==2.0.*
+    oldestdeps: pyerfa==2.0.1.1
```

Comment:
Why is the oldest version specified in such detail for `erfa`, but not for the other dependencies?

Reply:
pyerfa doesn't really follow semver.